php/commands/server.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class Server_Command extends WP_CLI_Command {
+
+	/**
+	 * Start a development server.
```

Comment:
Could we add a bit more description to how it actually works, usage details, and how you should and shouldn't use it?